recipes/SpatialPy/meta.yaml

```
@@ -0,0 +1,62 @@
+
+{% set name = "spatialpy" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4bf963ee707858dc8ff13b8f48dae18c4d6a0861965c525cc664ce098f676fd
+  
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+requirements:
+  build:
+    
+  host:
+    - python >3.5
+    - pip
+    - setuptools >=41.0.1
+    
+  run:
+    - python >3.5
+    - matplotlib-base >=3.0.3
+    - plotly >=4.1.0
+    - scipy >=1.4.1
+    - meshio >=5.3.2
+    - scons >= 4.4.0
+    - TODO: add Scons compatiable C++ compiler package here
```

```suggestion
    - {{ compiler('cxx') }}
```
This jinja template will be replaced by conda-build when the package is created with whatever platform-specific compiler is currently specified in our channel's global pinnings.

At runtime, I believe the path to this compiler is exported to the CXX environment variable.

https://conda-forge.org/docs/maintainer/pinning_deps.html#globally-pinned-packages
https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers